Sources/Instrumentation/Tracing/NoOpTracing.swift

```diff
@@ -52,15 +52,10 @@ public struct NoOpTracingInstrument: TracingInstrument {
 
         public mutating func addEvent(_ event: SpanEvent) {}
 
-        public var attributes: [String: SpanAttribute] {
-            [:]
-        }
-
-        public subscript(attributeName attributeName: String) -> SpanAttribute? {
+        public var attributes: SpanAttributes {
             get {
-                nil
-            }
-            set {
+                [:]
+            } set {
```

Comment:
weird formatting?

Reply:
I always used to do it like this as I picked it up somewhere (maybe even the Swift book, but not sure). I'm fine with putting it on the next line too.